ontology/uco/tool/tool.ttl

```diff
@@ -396,59 +425,229 @@ tool:compilers
 	.
 
 tool:cpeid
-	a owl:DatatypeProperty ;
+	a
+		owl:DatatypeProperty ,
+		owl:DeprecatedProperty
+		;
 	rdfs:label "cpeid"@en ;
 	rdfs:comment "Specifies the Common Platform Enumeration identifier for the software."@en ;
 	rdfs:range xsd:string ;
+	rdfs:seeAlso
+		observable:DeviceFacet ,
+		observable:SoftwareFacet ,
+		observable:cpeid
+		;
+	.
+
+tool:cpeid-subjects-shape
+	a sh:NodeShape ;
+	rdfs:seeAlso
+		observable:DeviceFacet ,
+		observable:SoftwareFacet ,
+		observable:cpeid
+		;
+	sh:property [
+		sh:maxCount "0"^^xsd:integer ;
+		sh:message "uco-tool:cpeid is deprecated.  Please instead use the path uco-core:hasFacet (linking a uco-observable:DeviceFacet or uco-observable:SoftwareFacet) / uco-observable:cpeid."@en ;
```

Comment:
I realized `:cpeid` is not associated with `observable:DeviceFacet`.  That should come in a separate proposal.